### ISSUE TITLE
Idea: automatically fetch more repos if the search result is empty

### ISSUE THREAD

**Issue by waldyrious** [korosuke613/gh-user-stars#6]
It's nice that by default this command is fast, by fetching only the 30 most recent stars; but it's inconvenient to type a search query in the interactive prompt, and if no match is found, having to exit it and start the command again with a custom, larger number parameter (which I won't even know what it should be).

In fact, I believe that searching for older starred repos is more likely to benefit from this helper, since for more recent repos, either I or my browser will probably have the repository name in the short-term memory :)

So it would be great if I could do `gh user-stars`, type something which results in no matches, and the tool would automatically fetch more results and match the query string against them.

In fact, ideally this background fetching should occur even if results _are_ found in the latest 30 stars, because it's possible for older stars to also match the query. If that extra fetching is done after the initial fetch, the current experience should not be affected and would remain relatively fast.

Is this something you could consider?

**Comment by korosuke613** [comment]
@waldyrious 
Thanks for the great idea!!
I would like to adopt the idea to keep fetching on the back side. I'm working on it now.

**Comment by korosuke613** [comment]
@waldyrious 
Changed to continuous fetch on the back side(#7). Is this the behavior you were expecting?

**Comment by waldyrious** [comment]
@korosuke613 the behavior shown in the gif at #7 certainly appears to be what I am looking for, but it doesn't seem to be working yet for me. I get the following error:

```
$ gh user-stars
> /home/waldyrious/.local/share/gh/extensions/gh-user-stars/libs.sh: line 32: jq: command not found
  /home/waldyrious/.local/share/gh/extensions/gh-user-stars/libs.sh: line 34: jq: command not found
  /home/waldyrious/.local/share/gh/extensions/gh-user-stars/libs.sh: line 35: jq: command not found

  0/0
```

I suppose I should install `jq`, which is now a requirement for this extension, just like `fzf`; but IMO it would be better if the tool failed more gracefully when the dependencies aren't found.

**Comment by korosuke613** [comment]
@waldyrious Oops... I hadn't noticed this. Thanks! Fixed #10 to add a message to make jq mandatory.

**Comment by waldyrious** [comment]
Nice! The error works well, and the continuous fetching does too. Thanks for the quick fix again!
